src/main/java/frc/robot/subsystems/Shooter/Shooter.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.subsystems.Shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Shooter extends SubsystemBase {
+    private TalonFX master = new TalonFX(Ports.Shooter.MASTER);
+    private VictorSPX slave1 = new VictorSPX(Ports.Shooter.SLAVE1);
+    private VictorSPX slave2 = new VictorSPX(Ports.Shooter.SLAVE2);
+    private final UnitModel UnitModel = new UnitModel(Constants.Shooter.TICKS_PER_ROTATION);
+
+    /**
+     * this constructor:
+     *  -sets motors inverted
+     *  -makes slaves to follow master
+     *  -sets PID constants
+     */
+    public Shooter(){

```

Comment:
Notice that when you are using TalonSrx you need to configure which sensor is connected.
see [https://www.ctr-electronics.com/downloads/api/java/html/classcom_1_1ctre_1_1phoenix_1_1motorcontrol_1_1can_1_1_talon_s_r_x.html#af2ef6fdd84542e78b1265baef5ddf328](url)

Reply:
And also use `setSensorPhase()`